controllers/fetchKeyController.ts

```diff
@@ -15,7 +15,7 @@ export const fetchKey = (req: Request, res: Response) => {
       });
   } else {
     return knex("settings")
-      .where("key", searchParams.key)
+      .where("key", "in", [searchParams.key])
```

Comment:
Can we merge both the queries into one "in" query? both are same at the moment

Reply:
Done.